contracts/RewardPool/modules/DelegationRewards.sol

```diff
@@ -403,6 +409,13 @@ abstract contract DelegationRewards is RewardPoolBase, Vesting, RewardsWithdrawa
 
     // _______________ Private functions _______________
 
+    function _changeMinDelegation(uint256 newMinDelegation) private {
+        if (newMinDelegation < 1 ether) { 
```

Comment:
Better have a MIN_STAKE_LIMIT constant instead of using hardcoded values

Reply:
added MIN_STAKE_LIMIT and MIN_DELEGATION_LIMIT constants